SiennaOpenAPIModels.jl/src/sienna_to_json/static_injections.jl

```diff
@@ -236,3 +236,36 @@ function psy2openapi(energy_res::PSY.EnergyReservoirStorage, ids::IDGenerator)
         dynamic_injector=getid!(ids, energy_res.dynamic_injector),
     )
 end
+
+function psy2openapi(hydro_res::PSY.HydroEnergyReservoir, ids::IDGenerator)
+    HydroEnergyReservoir(
+        id=getid!(ids, hydro_res),
+        name=hydro_res.name,
+        available=hydro_res.available,
+        bus=getid!(ids, hydro_res.bus),
+        active_power=hydro_res.active_power * PSY.get_base_power(hydro_res),
+        reactive_power=hydro_res.reactive_power * PSY.get_base_power(hydro_res),
+        rating=hydro_res.rating * PSY.get_base_power(hydro_res),
+        prime_mover_type=string(hydro_res.prime_mover_type),
+        active_power_limits=get_min_max(
+            scale(hydro_res.active_power_limits, PSY.get_base_power(hydro_res)),
+        ),
+        reactive_power_limits=get_min_max(
+            scale(hydro_res.reactive_power_limits, PSY.get_base_power(hydro_res)),
+        ),
+        ramp_limits=get_up_down(
+            scale(hydro_res.ramp_limits, PSY.get_base_power(hydro_res)),
+        ),
+        time_limits=get_up_down(hydro_res.time_limits),
+        base_power=hydro_res.base_power,
+        storage_capacity=hydro_res.storage_capacity * PSY.get_base_power(hydro_res),
+        inflow=hydro_res.inflow * PSY.get_base_power(hydro_res),
+        initial_storage=hydro_res.initial_storage * PSY.get_base_power(hydro_res),
+        operation_cost=get_hydrostorage_cost(hydro_res.operation_cost),
+        storage_target=hydro_res.storage_target,
+        conversion_factor=hydro_res.conversion_factor,
```

Comment:
Do you think we should be converting `conversion_factor`? The units look like p.u.-hour / m^3, so it might be worth scaling by base power to get units of MW hr / m^3.